net/seafile-server/Makefile

```diff
@@ -30,11 +29,10 @@ define Package/seafile-server
     CATEGORY:=Network
     TITLE:=Seafile server
     MAINTAINER:=Gergely Kiss <mail.gery@gmail.com>
-    URL:=http://seafile.com/
+    URL:=https://seafile.com/
     DEPENDS:=+libarchive +libopenssl +glib2 +libsearpc +seafile-ccnet +seafile-seahub +sqlite3-cli +python-mysql +python-urllib3 \
 		+jansson +libevent2 +libevent2-openssl +zlib +libzdb +libsqlite3 +libmysqlclient \
 		+libpthread +libuuid +bash +procps-ng +procps-ng-pkill +SEAFILE_FUSE_SUPPORT:libfuse $(ICONV_DEPENDS)
-    EXTRA_DEPENDS:=seafile-seahub (=6.3.0-d1ab146a936a6ea1e1581bf3f194e86742f0d3cd-1)
```

Comment:
My reasoning is that packages here tend not to use it. From a simple git grep, I see three others with one of them ripe for removal.

Reply:
I have added this line to ensure that versions of Seafile components are kept in sync by the package manager (it has no sane reason and also can cause issues if they are at different versions). Please update the version number as per #7381 (ie. set it to 6.3.4-1). The goal is to keep all Seafile packages (seafile-{server,seahub,ccnet}) at the same version at all times. I'd advise to update all three packages in one PR in the future to avoid issues about combining different versions. Note that libseafile is an exception as it doesn't have to be the same version as other components.

Reply:
I believe the change is good now.